vyper/builtins/_convert.py

```diff
@@ -403,56 +381,59 @@ def to_bytes_m(expr, arg, out_typ):
             arg = b.resolve(arg)
 
     elif is_bytes_m_type(arg.typ):
-        arg_info = arg.typ._bytes_info
         # clamp if it's a downcast
-        if arg_info.m > out_info.m:
-            arg = bytes_clamp(arg, out_info.m)
+        if arg.typ.m > out_typ.m:
+            arg = bytes_clamp(arg, out_typ.m)
 
-    elif is_integer_type(arg.typ) or is_base_type(arg.typ, "address"):
-        int_bits = arg.typ._int_info.bits
+    elif is_integer_type(arg.typ) or arg.typ == AddressT():
+        if arg.typ == AddressT():
+            int_bits = 160
```

Comment:
Is there a way to access this number from `AddressT` somehow?

Reply:
We thought about having `.bits` on AddressT, but ultimately decided not to since it's only used in one or two places. Also thought about having it inherit from uint160 but that also felt weird.